composer.json

```diff
@@ -27,7 +27,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "5.1.x-dev"
+            "dev-master": "5.2.x-dev"
```

Comment:
5.1 is unreleased so you can keep the alias as 5.1.x

Reply:
Right you are! Will update